segmenter/video_segmenter.go

```diff
@@ -82,7 +83,15 @@ func (s *FFMpegVideoSegmenter) RTMPToHLS(ctx context.Context, opt SegmenterOptio
 	tsfn := s.WorkDir + "/" + s.StrmID + "_%d.ts"
 
 	//This command needs to be manually killed, because ffmpeg doesn't seem to quit after getting a rtmp EOF
-	cmd := exec.Command("ffmpeg", "-i", s.LocalRtmpUrl, "-vcodec", "copy", "-acodec", "copy", "-bsf:v", "h264_mp4toannexb", "-f", "segment", "-muxdelay", "0", "-segment_list", plfn, tsfn)
+	glog.Infof("Ffmpeg path: %v", s.ffmpegPath)
+
+	var cmd *exec.Cmd
+	if s.ffmpegPath == "" {
+		cmd = exec.Command("ffmpeg", "-i", s.LocalRtmpUrl, "-vcodec", "copy", "-acodec", "copy", "-bsf:v", "h264_mp4toannexb", "-f", "segment", "-muxdelay", "0", "-segment_list", plfn, tsfn)
+	} else {
+		cmd = exec.Command(s.ffmpegPath+"/ffmpeg", "-i", s.LocalRtmpUrl, "-vcodec", "copy", "-acodec", "copy", "-bsf:v", "h264_mp4toannexb", "-f", "segment", "-muxdelay", "0", "-segment_list", plfn, tsfn)
```

Comment:
should probably use `path.Join()` for cross platform compatibility and for handling the extra/missing `/` case